robolectric/src/main/java/org/robolectric/junit/rules/ExpectedLogMessagesRule.java

```diff
@@ -78,8 +78,6 @@ public final class ExpectedLogMessagesRule implements TestRule {
           "RingtoneManager",
           // Fails when attempting to preload classes by name
           "PhonePolicy",
-          // Ignore MultiDex log messages
```

Comment:
Can this be removed? Although multidex is a no-op for L and above, many teams still use it, and they will still see log messages:

https://cs.android.com/android/platform/superproject/main/+/main:frameworks/multidex/library/src/androidx/multidex/MultiDex.java;l=96?q=multidex.java

Reply:
You're right. I've reverted this change.